src/components/Header.jsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import { Link } from 'react-router-dom';
+import { Menu, Segment } from 'semantic-ui-react'
+
+const Header = () => {
+	return (
+		<Segment inverted>
+			<Menu inverted secondary>
+        <Menu.Item 
+          data-cy="header-news"
+          as={Link}
+          to="/">
+					News
+				</Menu.Item>
+        <Menu.Item
+          data-cy="header-gothenburg"
+          to="/"
+          as={Link}>
+					Gothenburg
+				</Menu.Item >
+        <Menu.Item
+          data-cy="header-economy"
+          to="/"
+          as={Link}>
+					Economy
+				</Menu.Item >
+        <Menu.Item
+          data-cy="header-politics"
+          to="/"
+          as={Link}>
+					Politics
+				</Menu.Item >
+			</Menu>
+		</Segment>
+	);
+};
+
+export default Header;
```

Comment:
please format this file and make sure that to use indentation using spaces instead of tabs. 
Also, I think you should definitely have a header, but adding a lot of buttons that leads to the same place is not very user friendly. Please stick to one button that can tak the user back to all the articles until you add categories

Reply:
Fixed it up!